merkle-js/src/ec_stark.ts

```diff
@@ -0,0 +1,26 @@
+import * as starkCurve from 'micro-starknet';
+import { merkle, num,cairo, encode } from 'starknet';
+
+// array of  [address, token_id, task_id, name, rank, score, level, total_eligible_users]
+let list = [['0x35742331cbd91fc4873c469ba52939db909efcf46b9c28e82d1baaa7078a6d6', 10,1,'L1PW',10,406872312314,0,76769]];
+const private_key = '0x019800ea6a9a73f94aee6a3d2edf018fc770443e90c7ba121e8303ec6b349279';
```

Comment:
Is this private key supposed to be committed?

Reply:
it's just for testing